src/index.js

```diff
@@ -1,5 +1,5 @@
 export ActionPrototype from './lib/ActionPrototype'
 export ExtraStorage from './lib/extra_storage'
 export Dispatcher from './lib/fu-dispatcher'
-export JsonFetcher from './lib/jsonFetcher'
+export JsonStatham from './lib/jsonStatham'
```

Comment:
@maneeshnycda Inconsistent capitalization. It should always be camelCase not PascalCase since we don't call `new` before it.